examples/morning_serenity.rb

```diff
@@ -0,0 +1,27 @@
+require 'shoes'
+require 'bloops'
```

Comment:
Probably gonna wanna put this in the bloopsaphone examples dir so CI doesn't try to run it.

Reply:
Good point. thanks! and I support the requires aren't required